framework/include/cppmicroservices/BundleContext.h

```diff
@@ -259,12 +265,9 @@ class US_Framework_EXPORT BundleContext
    *         registering the service to update the service's properties or to
    *         unregister the service.
    *
-   * @throws std::invalid_argument If one of the following is true:
-   *         <ul>
-   *         <li><code>service</code> is <code>0</code>.
-   *         <li><code>properties</code> contains case variants of the same key name.
-   *         </ul>
-   * @throws std::logic_error If this BundleContext is no longer valid.
+   * @throws std::runtime_error If this BundleContext is no longer valid.
```

Comment:
A `std::runtime_error` will also be thrown if there are case variants of the same key in the supplied ` properties` map.